experiment/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- aojea
+approvers:
+- aojea
```

Comment:
I don't know what is the convention for new files, so I started small adding only myself,